bucket/cloc.json

```diff
@@ -2,7 +2,7 @@
     "homepage": "https://github.com/AlDanial/cloc",
     "description": "Counts blank lines, comment lines, and physical lines of source code in many programming languages",
     "version": "1.82",
-    "license": "GPL-2.0",
+    "license": "GPL-2.0-only|Artistic-2.0",
```

Comment:
Decide which format you want to prefer and then use it everywhere.

You are mixing `,` and `|` without any pattern or anything.

Reply:
> If multiple licenses, seperate licenses with a comma (,). If dual licensed, seperate licenses with a pipe symbol (|).

I just follow the doc.